README.md

```diff
@@ -355,37 +355,55 @@ set AZURITE_ACCOUNTS="account1:key1:key2;account2:key1:key2"
 
 ### HTTPS Setup
 
-#### Mkcert
+#### PEM
 
-You can generate a local certificate for 127.0.0.1 with `mkcert`. See the mkcert GitHub repo for [installation instructions](https://github.com/FiloSottile/mkcert), and then run the following commands.
+First, you need download `choco` by run the following commands in PowerShell.
```

Comment:
For the installation, it is okay to link to the site, because you don't know what OS the person is using: https://github.com/FiloSottile/mkcert#installation